### ISSUE TITLE
Dota 2 sound crackling 

### ISSUE THREAD

**Issue by YouYoud2** [ValveSoftware/Dota2-Gameplay#18964]
### Description

Hi,

since this morning, my sound has been cracking randomly without doing anything in Dota 2 every 1-15 minutes especially during teamfight (and only in Dota 2). So I've tried to :

- Reboot computer
- Update my driver from my soundboard card (Yamaha AG03)
- Update my driver from my gpu
- Verify integrity of game files
- Reinstall dota 2
- Change sound device in Dota
- Change the frequency in Windows
- Directx => Vulkan
- With or w/o nvidia broadcasting
- Trying to find the source of the crackling sound by using low and high frequency sound
- Checking Windows events viewer => Nothing
- My CG, proc & ram never get overwhelmed
- Did sfc /scannow it repaired corrupted files

Here an exemple, I can give you a lot more if you need :

https://clips.twitch.tv/ArbitraryGeniusGazelleFeelsBadMan-MHpRp0TbM9wliG5w

Hope you can help, I don't know what to do...

edit : -directsound in launch settings fix the problem ?

**Comment by kaffle** [comment]
also having this issue. it mimics the quality you might have with sample rate mismatch, but it's only for a few seconds at a time

**Comment by dasfuu** [comment]
Having the same problem since yesterday. Sounds aweful.

**Comment by EmilianoTalamo** [comment]
Same here, seems to be happening on high sample rate interfaces or DACs.

https://www.reddit.com/r/DotA2/comments/1d44udf/anyone_having_sound_issues_or_any_clue_how_to_fix/

https://www.reddit.com/r/DotA2/comments/1d4bqe3/dota_2_sound_crackling_since_this_morning/

This issue is only happening in Dota since yesterday. Any other audio in parallel (like Twitch or Spotify) is ok.

Switching to another audio output and then switching back fixes this issue for a couple of minutes.

**Comment by gsotiriou** [comment]
This is an issue indeed.

**Comment by papahuge** [comment]
Yeah same thing here, I'm using the RME AIO Pro audio interface that has very good solid drivers and high sampling rate capabilities. Since a couple days ago the audio in DOTA2 and only in DOTA2 has started to pop and clip and crackle. It's more noticeable at 48khz rate, and if I switch to 44.1kHz it doesn't do it as much but still happens. Changing buffer rate doesn't really fix anything beside creating more latency delay. 

This def has something to do with the latest update.

**Comment by ani917** [comment]
started with the last update

**Comment by urbeastfriend** [comment]
Same since last update, only in dota. Seems to happen more often for users with dac. Nothing helps. Its terrible experience

**Comment by bbyc4kes** [comment]
**same thing.**
sizzling and cracking sounds appear every 5-7 mins and last for the same amount of time, especially high rate of occurring with a lot of sounds layering up

**Comment by smi42** [comment]
started yesterday after the update having the same issue

**Comment by gsotiriou** [comment]
Latest mini update seems to have fixed it for me.

**Comment by papahuge** [comment]
It's happening again!! It was fixed before the recent 3rd Crownfall act update. But now its all crackling pretty hard.

**Comment by bbyc4kes** [comment]
the only thing that seems to help me at the time on my Windows 11, Logitech G733 is to go to Device Manager > Sound, video and game controllers , select your output device that you are using in Dota, go to > Driver > Uninstall device, and then reboot the machine.

the only issue with this solution is that it only helps for a certain period of time and you have to perform the same strategy over and over again every time the problem pops up again.

i also noticed that the issue is more highly to appear if you have changed your output device on the computer throughout your session. for example, if you used your speaker as an output device for some time and then you connected your headset and decided to use it, its almost guaranteed that the cracking sound will appear, at least in my case.

**Comment by papahuge** [comment]
> the only thing that seems to help me at the time on my Windows 11, Logitech G733 is to go to Device Manager > Sound, video and game controllers , select your output device that you are using in Dota, go to > Driver > Uninstall device, and then reboot the machine.
> 
> the only issue with this solution is that it only helps for a certain period of time and you have to perform the same strategy over and over again every time the problem pops up again.
> 
> i also noticed that the issue is more highly to appear if you have changed your output device on the computer throughout your session. for example, if you used your speaker as an output device for some time and then you connected your headset and decided to use it, its almost guaranteed that the cracking sound will appear, at least in my case.

also you can try to change the sample rate of your audio card driver from 48kHz to 44.1kHz this sometimes helps.

**Comment by bbyc4kes** [comment]
> > the only thing that seems to help me at the time on my Windows 11, Logitech G733 is to go to Device Manager > Sound, video and game controllers , select your output device that you are using in Dota, go to > Driver > Uninstall device, and then reboot the machine.
> > the only issue with this solution is that it only helps for a certain period of time and you have to perform the same strategy over and over again every time the problem pops up again.
> > i also noticed that the issue is more highly to appear if you have changed your output device on the computer throughout your session. for example, if you used your speaker as an output device for some time and then you connected your headset and decided to use it, its almost guaranteed that the cracking sound will appear, at least in my case.
> 
> also you can try to change the sample rate of your audio card driver from 48kHz to 44.1kHz this sometimes helps.

ive also found out that just unplugging and plugging back in output device sometimes helps

**Comment by ArekPiekarz** [comment]
I also have this issue with hi-fi DACs - Ferrum Wandla and Schiit Gunnr. I tried all combinations of bit depths and sample rates, they all have crackling. For me the only working hardware are Fiio BTR15 (a portable bluetooth DAC/Amp, in PC mode) and a headset Razer Blackshark V2 Pro 2023. Valve, please try to fix this!

**Comment by ArekPiekarz** [comment]
I found that this is caused by Steam's game recording feature - https://store.steampowered.com/gamerecording. When I disabled it, the crackling stopped. However, that functionality is very useful and I would like to be able to use both the external DAC and the recording at the same time.
